web/checks_api.go

```diff
@@ -100,17 +122,38 @@ func ApiCreateChecksCatalogHandler(s services.ChecksService) gin.HandlerFunc {
 // @Success 200 {object} JSONChecksSettings
 // @Failure 404 {object} map[string]string
 // @Router /api/checks/{id}/settings [get]
-func ApiCheckGetSettingsByIdHandler(s services.ChecksService) gin.HandlerFunc {
+func ApiCheckGetSettingsByIdHandler(consul consul.Client, s services.ChecksService) gin.HandlerFunc {
```

Comment:
Now, this time,  I will play the devil advocate game XD
Why don't you use the new discovered data here instead of consul data?
We already have all the cloud data in the database. This is actually by now outdated.

Anyway, we can refactor at any time.

Reply:
And you're actually right! I wanted to wait until we had the hosts service in place too, either we wait until the host service hits `main` or we go live with this and refactor later as it shouldn't be a big deal. It's ok for me to wait, anyway :smile:

Reply:
As commented here https://github.com/trento-project/trento/pull/363#discussion_r742731024 we can take different design decisions (load the things from other read models, project the whole connection settings, etc).
Anyway, meanwhile we can move forward with this thing, knowing that it needs some work later.